src/dom/DOMPipe.ts

```diff
@@ -122,7 +122,10 @@ export class DOMPipe implements RenderPipe<DOMContainer>
             canvas.parentNode?.appendChild(this._domElement);
         }
 
-        this._domElement.style.transform = `translate(${canvas.offsetLeft}px, ${canvas.offsetTop}px)`;
+        const scale = (parseFloat(canvas.style.height) / canvas.height) * this._renderer.resolution;
+
+        // scale according to the canvas scale and translate
+        this._domElement.style.transform = `translate(${canvas.offsetLeft}px, ${canvas.offsetTop}px) scale(${scale})`;
```

Comment:
Do we not need to also check the width here?

```ts
const scaleX = (parseFloat(canvas.style.width) / canvas.width) * this._renderer.resolution;
const scaleY = (parseFloat(canvas.style.height) / canvas.height) * this._renderer.resolution;

// scale according to the canvas scale and translate
this._domElement.style.transform = `translate(${canvas.offsetLeft}px, ${canvas.offsetTop}px) scale(${scaleX}, ${scaleY})`;

Reply:
we 100% should !